lib/pages/photo_page.dart

```diff
@@ -58,14 +61,31 @@ class _PhotoPageState extends State<PhotoPage> {
   }
 
   Future<void> confirmPhoto() async {
-    final success =
-        await GallerySaver.saveImage(file.path, albumName: 'Lifeance');
-    if (success ?? false) { 
-    final directory = await getExternalStorageDirectory();
-    final savedImagePath = '${directory?.path}/Lifeance/${file.path.split('/').last}';
-    print('Image saved to $savedImagePath');
+    final directory = await getApplicationDocumentsDirectory();
+    final savedImagePath =
+        '${directory.path}/Lifance/${file.path.split('/').last}';
```

Comment:
podsložku `Lifeance` si vytvářet nemusíš. Spíš klidně něco jako `/challengePhotos/` by se hodilo víc. Ono totiž `getApplicationDocumentsDirectory` ti už samo vrátí cestu do složky vyhrazené čistě pro tvoji aplikaci. Třeba na androidu je to tuším někde ve složce android/com.super.app (bundle ID)